src/components/Chat.js

```diff
@@ -73,6 +74,54 @@ function Chat() {
     []
   );
   const messages = useFirebaseQuery(messagesQuery);
+  const [lastMsg, setLastMsg] = useState("");
+  const [msgCounter, setMsgCounter] = useState(0);
+
+  // eslint-disable-next-line no-unused-vars
+  const [previousMessages, setPreviousMessages] = useState({});
```

Comment:
This variable is unused. Rather than disabling the lint, you could also just remove the variable entirely.